tests/humanizer.spec.ts

```diff
@@ -53,3 +54,22 @@ describe('humanizer', () => {
     expect(result).to.equal('Unknown');
   });
 });
+
+describe('risk levels', () => {
+  it('Should give None risk level when score is below 0', () => {
+    expect(toSeverity(0)).to.equal('None');
+    expect(toSeverity(-7)).to.equal('None');
+  });
+  it('Should give Low risk level when score is below 3', () => {
+    expect(toSeverity(3)).to.equal('Low');
+    expect(toSeverity(1.2654)).to.equal('Low');
+  });
+  it('Should give Medium risk level when score is below 6', () => {
+    expect(toSeverity(6)).to.equal('Medium');
+    expect(toSeverity(4.2654)).to.equal('Medium');
+  });
+  it('Should give High risk level when score is below 8.5', () => {
+    expect(toSeverity(8.5)).to.equal('High');
+    expect(toSeverity(7.2654)).to.equal('High');
+  });
+});
```

Comment:
Suggested tests changes:
- move them to existing humanizer test suite
- wording changes
- tests for boundary values
- tests for "Critical" qualitative severity rating
- test only valid score values

```suggestion

  it('should humanize score as "None" for zero value', () => {
    expect(humanizeScore(0)).to.equal('None');
  });

  it('should humanize score as "Low" for values from interval [0.1, 3.9]', () => {
    expect(humanizeScore(0.1)).to.equal('Low');
    expect(humanizeScore(1.2)).to.equal('Low');
    expect(humanizeScore(3.9)).to.equal('Low');
  });

  it('should humanize score as "Medium" for values from interval [4.0, 6.9]', () => {
    expect(humanizeScore(4.0)).to.equal('Medium');
    expect(humanizeScore(4.2)).to.equal('Medium');
    expect(humanizeScore(6.9)).to.equal('Medium');
  });

  it('should humanize score as "High" for values from interval [7.0, 8.9]', () => {
    expect(humanizeScore(7.0)).to.equal('High');
    expect(humanizeScore(7.5)).to.equal('High');
    expect(humanizeScore(8.9)).to.equal('High');
  });

  it('should humanize score as "Critical" for values from interval [9.0, 10.0]', () => {
    expect(humanizeScore(9.0)).to.equal('Critical');
    expect(humanizeScore(9.5)).to.equal('Critical');
    expect(humanizeScore(10.0)).to.equal('Critical');
  });
});
```